core/color/partners.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
these are the colours required, as defined here: https://www.figma.com/file/n9T5usku57ecvPArT9SpWO/Tokens---Core-Color?node-id=3906%3A2&t=ZYuvVMkkHBjPHmSI-0